integrated_channels/integrated_channel/learner_data.py

```diff
@@ -25,26 +25,30 @@ class BaseLearnerExporter(object):
     Base class for exporting learner completion data to integrated channels.
     """
 
+    GRADE_PASSING = 'Pass'
+    GRADE_FAILING = 'Fail'
+    GRADE_INCOMPLETE = 'In Progress'
+
     @property
     def grade_passing(self):
         """
         Returns the string used for a passing grade.
         """
-        return 'Pass'
+        return self.GRADE_PASSING
```

Comment:
Okay, that makes sense! As with above, let's leave that alone for right now, then, and we can revisit as needed. I'm going to go ahead and do some testing now. :)